src/network.zig

```diff
@@ -1027,15 +1027,16 @@ pub const Protocols = struct {
 						},
 					}
 				},
-				type_reserved3 => {},
-				type_reserved4 => {},
-				type_reserved5 => {},
-				type_reserved6 => {},
-				type_timeAndBiome => {
+				.reserved3 => {},
+				.reserved4 => {},
+				.reserved5 => {},
+				.reserved6 => {},
+				.timeAndBiome => {
 					if(conn.manager.world) |world| {
-						const zon = ZonElement.parseFromString(main.stackAllocator, null, reader.remaining);
-						defer zon.deinit(main.stackAllocator);
-						const expectedTime = zon.get(i64, "time", 0);
+						const expectedTime = try reader.readInt(i64);
+						const biomeIdLen = try reader.readInt(usize);
+						const biomeId = try reader.readSlice(biomeIdLen);
```

Comment:
Why not just send the biome palette id?
(which didn't exist when I originally wrote this)

Reply:
Are block and biome palettes synchornized?

Reply:
yes, they are sent together with the world assets.

Reply:
Cool